cmd/agent/main.go

```diff
@@ -86,6 +86,8 @@ func main() {
 	}
 
 	if cfg.Integrations.Enabled {
+		cfg.Integrations.ClientCA = cfg.Server.HTTPTLSConfig.ClientCAs
+		cfg.Integrations.ClientAuthType = cfg.Server.HTTPTLSConfig.ClientAuth
```

Comment:
from offline discussion: let's stop syncing these since the server settings and client settings can conflict.